addons/skin.estuary/xml/DialogPVRInfo.xml

```diff
@@ -52,6 +52,15 @@
 					<label>$INFO[ListItem.ChannelName,[B],[/B][CR]]$INFO[ListItem.Date,[COLOR grey]$LOCALIZE[552]:[/COLOR] ,[CR]]$INFO[ListItem.Duration,[COLOR grey]$LOCALIZE[180]:[/COLOR] ,[CR]]$VAR[PremieredLabel]$INFO[ListItem.Rating,[COLOR grey]$LOCALIZE[563]:[/COLOR] ,[CR]]$VAR[ExpirationDateTimeLabel]$INFO[ListItem.Genre,[COLOR grey]$LOCALIZE[515]:[/COLOR] ,[CR]]$INFO[ListItem.Writer,[COLOR grey]$LOCALIZE[20417]:[/COLOR] ,[CR]]$INFO[ListItem.Director,[COLOR grey]$LOCALIZE[20339]:[/COLOR] ,[CR]]$INFO[ListItem.Cast,[COLOR grey]$LOCALIZE[206]:[/COLOR] ,[CR]][CR]$INFO[ListItem.Plot]</label>
 					<autoscroll time="3000" delay="4000" repeat="5000">Skin.HasSetting(AutoScroll)</autoscroll>
 				</control>
+				<control type="textbox" id="400">
+					<left>660</left>
+					<top>20</top>
+					<width>1050</width>
+					<height>425</height>
+					<align>right</align>
+					<label>$VAR[FlagLabel]</label>
+					<autoscroll time="3000" delay="4000" repeat="5000">Skin.HasSetting(AutoScroll)</autoscroll>
+				</control>				
```

Comment:
1.) there's no need to use a 1050x425 scrolling textbox just to display a small label ;-)
a small sized label control would be more suitable.
also, there's no need to add an id to the control.

2.) this will overlap with the plot once the plot starts scrolling
![overlap](https://user-images.githubusercontent.com/687265/77016637-8c5b3100-6978-11ea-9381-b44a43118ae0.jpg)

Reply:
On 1): done. On 2) I'm not sure how to make this part of the autoscroll area. Open to suggestions on how to do this.

Reply:
Ok, thanks to an idea from @jjd-uk I moved the text to the SesonEpisode label. Looks and fits much better.

Reply:
<img width="2146" alt="Screenshot 2020-03-19 at 11 48 40" src="https://user-images.githubusercontent.com/4601187/77064558-982a1000-69d7-11ea-909b-72579ec88c39.png">

Reply:
If you ask me, the "New" flag feels misplaced in front of the episode/season number. I really looks like somebody was looking for free space on the screen and found exactly this place. ;-)

Reply:
Just my 2ct, though